extras/pahma/app/views/catalog/_slideshow_modal.html.erb

```diff
@@ -0,0 +1,15 @@
+  <!-- Modal -->
```

Comment:
Overwriting this file to add aria attributes (formerly done via blacklight_gallery.js) and make the #documents element focusable.